source/_components/bme680.markdown

```diff
@@ -114,7 +114,7 @@ aq_humidity_bias:
   default: 25
   type: integer
 temp_offset:
-  description: "The temperature for the sensor will always be too high as it pulls heat from the components around it.  Consider adding a negative offset to ensure the sensor returns an accurate temperature. Note: This value is in celsius."
+  description: "The temperature for the sensor will always be too high as it pulls heat from the integrations around it.  Consider adding a negative offset to ensure the sensor returns an accurate temperature. Note: This value is in celsius."
```

Comment:
This is incorrect in this context and should not be replaced. It is about measuring temperatures for physical components around the actual sensor.

Reply:
I have undone the adjustment 😉